firmware/quintuna/VC/src/app/app_efuse.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include <stdbool.h>
+#include "io_efuse.h"
+
+void app_efuse_init(
+    void (*const enabled_setters[NUM_EFUSE_CHANNELS])(bool),
+    void (*const current_setters[NUM_EFUSE_CHANNELS])(float));
```

Comment:
init is a deprecated idea. please localize all data into the corresponding modules, import, or heaven forbid extern (but have a good reason to do so)